etc/linux-desktop/syncthing-start.desktop

```diff
@@ -2,7 +2,7 @@
 Name=Start Syncthing
 GenericName=File synchronization
 Comment=Starts the main syncthing process in the background.
-Exec=/usr/bin/syncthing -no-browser -logfile=~/.config/syncthing/syncthing.log
+Exec=/usr/bin/syncthing -no-browser -logfile=""
```

Comment:
This is a really surprising way to say "logfile in the standard location" :/

Reply:
Yes it is. Unfortunately go doesn't allow string flags with a default value for when the flag is used, but no argument given, so that's that. The alternative would be to add `-logfile-default`, but that's ugly too.

Reply:
I think it's fair to say that this is undiscoverable to the point of being useless, really. Do we need to have the "logfile, but I'm not going to say where, just put it at the default" functionality on Unix?

That whole "the strings are too long so we default  to blank and then fix it up later" might be causing more trouble than it's worth, too.

Reply:
I believe it's worth it. For example in case of DE autostarts through this desktop file, which I think is a very likely first contact point for a "non-technical" user when trying to setup something to run at start. I'd add another magic value `default` alongside `-` instead of an empty string. And maybe add a short explanation in the extended usage about logging. Does that sound more useful/less obscure?

Reply:
Yeah, I think so.